### DISCUSSION TITLE
If I want to use my own data, do I only need to generate coordinates without resampling?

### DISCUSSION THREAD

**Discussion by yexiao11** [Project-MONAI/tutorials#1611] [General]
For the detection task about LUNA16, you said : "We follow the official 10-fold data splitting from the LUNA16 challenge and generate data-split JSON files using the script from [nnDetection]. In these files, the values of "box" are the ground truth boxes in world coordinates." Are these coordinates resampled to the same pixel spacing? If I want to use my own data, do I only need to generate coordinates without resampling?

**Comment by KumoLiu** [comment]
Hi @yexiao11, yes, and if you want to resample you may need to update the transform chain. Thanks!

**Reply by yexiao11** [reply]
If I want to use labels in image coordinates directly, then I just need to remove the "AffineBoxToImageCoordinates" step in the data preprocessing. Is that correct? Thanks!

**Reply by yexiao11** [reply]
But I get warnings about some image: Num foregrounds 0, Num backgrounds 70482500, unable to generate class balanced samples, setting `pos_ratio` to 0. Maybe my data is not correct. How can I check my bbox easily? Thanks!

**Reply by KumoLiu** [reply]
`AffineBoxToImageCoordinate` is used to convert a box in the world coordinate to the image coordinate.
https://github.com/Project-MONAI/MONAI/blob/e1ffa7ebd38963bdced326d86979f5a9e2851517/monai/apps/detection/transforms/dictionary.py#L257

You can refer to this tutorial.
https://github.com/Project-MONAI/tutorials/tree/main/detection/luna16_visualization
Thanks.
